src/modules/emote_menu/stores/emoji-categories.js

```diff
@@ -0,0 +1,125 @@
+import twemoji from 'twemoji';
+import emojiBlacklist from '../../../utils/emoji-blacklist.js';
+import Emote from '../../emotes/emote.js';
+import Icons from '../components/Icons.jsx';
+// file gets created during bundle
+// eslint-disable-next-line import/no-unresolved
+import emojiByCategory from './emojis-by-category.json';
+
+function countEmojis(emoji) {
+  let count = 0;
+  twemoji.parse(emoji.char, (d) => {
+    count += d.split('-').length;
+  });
+  return count;
+}
+
+function parseEmotes(emojiBySlug, provider) {
+  return Object.values(emojiBySlug)
+    .filter((emoji) => emojiBlacklist.indexOf(emoji.char) === -1 && countEmojis(emoji) === 1)
+    .map((emoji, index) => {
+      let url;
+
+      twemoji.parse(emoji.char, {
```

Comment:
we should avoid loading these twice, as it's a lot of additional parsing. is there a way to deduplicate this work?